tw-tasks-core/src/main/java/com/transferwise/tasks/handler/SimpleTaskProcessingPolicy.java

```diff
@@ -37,6 +39,11 @@ public class SimpleTaskProcessingPolicy implements ITaskProcessingPolicy {
   @Accessors(chain = true)
   private StuckTaskResolutionStrategy stuckTaskResolutionStrategy = StuckTaskResolutionStrategy.MARK_AS_ERROR;
 
+  @Getter
+  @Setter
+  @Accessors(chain = true)
+  private IPartitionKeyStrategy partitionKeyStrategy = new RandomPartitionKeyStrategy();
```

Comment:
This way we can not utilize the default bean you defined in the auto config.
I would leave it null here.
And then in KafkaTasksExecutionTriggerer utilize the default bean for it as fallback mechanism.

Reply:
Done